python/lsst/ts/ofc/scripts/generate_intrinsic_zernikes.py

```diff
@@ -0,0 +1,163 @@
+# This file is part of ts_ofc.
+#
```

Comment:
I think this file should be under:

"""
ts_ofc:
   --python/
   --scripts/
   ----generate_intrinsic_zernikes.py
"""